src/type/textCore.js

```diff
@@ -95,59 +521,536 @@ function textCore(p5, fn) {
    */
 
   /**
-   * TODO
+   * Sets the font used by the <a href="#/p5/text">text()</a> function.
+   *
+   * The first parameter, `font`, sets the font. `textFont()` recognizes either
+   * a <a href="#/p5.Font">p5.Font</a> object or a string with the name of a
+   * system font. For example, `'Courier New'`.
+   *
+   * The second parameter, `size`, is optional. It sets the font size in pixels.
+   * This has the same effect as calling <a href="#/p5/textSize">textSize()</a>.
+   *
+   * Note: `WEBGL` mode only supports fonts loaded with
+   * <a href="#/p5/loadFont">loadFont()</a>.
    *
    * @method textFont
+   * @return {Object} current font or p5 Object.
```

Comment:
Should this be a parameter instead?
```suggestion
   * @param {p5.Font|String|Object} font The font to apply
```